src/test/java/com/aws/greengrass/util/platforms/windows/WindowsPlatformTest.java

```diff
@@ -198,7 +203,7 @@ void GIVEN_file_system_permission_WHEN_convert_to_acl_THEN_succeed() throws IOEx
         int everyoneAclCount = 0;
         for (AclEntry aclEntry : initialOwnerAcl.getAcl()) {
             String name = aclEntry.principal().getName();
-            if (name.contains("Everyone")) {
+            if (name.contains(EVERYONE_GROUP_NAME)) {
```

Comment:
should this be `name.equals(EVERYONE_GROUP_NAME)`? could you create a group name like "Fake Everyone" and have this check pass?

Reply:
This is in our test code so it shouldn't really matter at all. But yeah, that is a reasonable change.

Reply:
Updated. Here `name` will actually equals "\Everyone"